src/javascript/js-webshim/minified/polyfiller.init.js

```diff
@@ -0,0 +1,6 @@
+(function($) {
+    $.webshim.bugs.bustedValidity = !('minLength' in document.createElement('input'));
+    $.webshims.setOptions('basePath', Zikula.Config.baseURL+'javascript/js-webshim/minified/shims/'); 
+    $.webshims.activeLang(Zikula.Config.lang);
+    $.webshims.polyfill(Zikula.Config.polyfillFeatures);
+})(jQuery);
```

Comment:
Yes, both files are needed. It's minified, because it loads the minified shims at "javascript/js-webshim/minified/shims/". The unminified shims are loaded at "javascript/js-webshim/dev/shims".